velox/exec/tests/utils/PlanBuilder.h

```diff
@@ -477,12 +477,18 @@ class PlanBuilder {
       bool ignoreNullKeys,
       const std::vector<TypePtr>& resultTypes = {});
 
-  /// Add a GroupIdNode using the specified grouping sets, aggregation inputs
-  /// and a groupId column name. And create GroupIdNode plan node with grouping
+  /// Add a GroupIdNode using the specified grouping sets, aggregation inputs,
+  /// grouping keys and a groupId column name.
+  /// The extraGroupingKeys specify the additional projections of output
+  /// key columns to the input columns. This allows grouping sets with
+  /// multiple groups corresponding to the same input key.
+  /// Note: Grouping sets are specified using output key columns.
```

Comment:
This is very confusing. 

My understanding is GroupIdNode effectively includes an optional projection that may duplicate and/or rename some input columns.

Reply:
Thanks. I've refactored the API.